### PR TITLE
Remove comments about complex selector

### PR DESCRIPTION
**Pull request by Konrud**
I think it's time we could remove those comments, at least from the Example. According to [caniuse](https://caniuse.com/?search=not) it is pretty well supported.

As regarding the point in the description (i.e. _Using two selectors at the same time might not be supported in all browsers. Example: `:not(.foo, .bar)`. For wider support you could use, `:not(.foo):not(.bar)`_) 
I believe we can leave it for now, maybe changing it a bit to point out that for the browsers that still don't support it one could use the following syntax.
Would be happy to know other opinions.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/_colon_not/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ There are several unusual effects and outcomes when using `:not()` that you shou
 - This pseudo-class can increase the [specificity](/en-US/docs/Web/CSS/Specificity) of a rule. For example, `#foo:not(#bar)` will match the same element as the simpler `#foo`, but has a higher specificity.
 - `:not(.foo)` will match anything that isn't `.foo`, _including {{HTMLElement("html")}} and {{HTMLElement("body")}}._
 - This selector only applies to one element; you cannot use it to exclude all ancestors. For instance, `body :not(table) a` will still apply to links inside of a table, since {{HTMLElement("tr")}} will match with the `:not()` part of the selector.
-- Using two selectors at the same time is not yet supported in all browsers. Example: `:not(.foo, .bar)`. For wider support you could use, `:not(.foo):not(.bar)`
+- Using two selectors at the same time might not be supported in all browsers. Example: `:not(.foo, .bar)`. For wider support you could use, `:not(.foo):not(.bar)`
```
Comment (teoli2003) [inline files/en-us/web/css/_colon_not/index.md:43]: I think we can even get one step further and change the whole sentence
```suggestion
- You can negate several selectors at the same time. Example: `:not(.foo, .bar)` is equivalent to `:not(.foo):not(.bar)`.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/_colon_not/index.md (hunk 1 above)
Comment (Konrud) [reply files/en-us/web/css/_colon_not/index.md:43]: Splendid idea. Thank you.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

**Pull request merged**